### ISSUE TITLE
M3U VLC Export

### ISSUE THREAD

**Issue by dreamflasher** [mifi/lossless-cut#2051]
### The fewer issues I have to read, the more new features I will have time to implement, so I ask that you please try these things first

- [X] Check out all the menus and keyboard shortcuts. Maybe the feature already exists?
- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest)
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md) to see if the feature already exists
- [X] See if someone already suggested this under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)

### Description

I'd like to be able to export projects as M3U playlist files, including vlc start-stop info:

```
#EXTVLCOPT:start-time=28.3
#EXTVLCOPT:stop-time=36.7
video.mp4
```

This would allow me to play cut videos without storing the cut segments (and thus saving disk space) and allows lossless cutting on a frame level.

**Comment by mifi** [comment]
Interesting use case. So you want to be able to store an m3u playlist which has a bunch of start-stop times for the same video file, thus when a player plays it back, the player effectively just seeks back and forth in the same video?

**Comment by dreamflasher** [comment]
Yeah, meanwhile I actually implemented a python script to convert LLC to M3U, maybe it's helpful for others or could be used as an inspiration:

```
import urllib.parse
from pathlib import Path

import demjson3
from fire import Fire


def main(llc: Path):
    llc = Path(llc)
    l = demjson3.decode(llc.read_text())
    segments = l["cutSegments"]
"cooldown"]
    for i, segment in enumerate(segments):
        out = ["#EXTM3U"]
        out.append(f"#EXTVLCOPT:start-time={segment['start']}")
        out.append(f"#EXTVLCOPT:stop-time={segment['end']}")
        out.append("../" + urllib.parse.quote(l["mediaFileName"]))
        Path(llc.parent / f"{llc.stem}-{i}-{segment['name']}.m3u").write_text("\n".join(out))


if __name__ == '__main__':
    Fire(main)
```

I'm using this with M3U, but it also works with M3U8, I am not attached to the file extension.

**Comment by mifi** [comment]
thanks. i agree it's a cool feature so i'll consider implementing it
